proto/viam/service/worldobjectstore/v1/world_object_store.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package viam.service.worldobjectstore.v1;
+
+import "common/v1/common.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/struct.proto";
+
+option go_package = "go.viam.com/api/service/worldobjectstore/v1";
+option java_package = "com.viam.service.worldobjectstore.v1";
+
+service WorldObjectStoreService {
+  // ListWorldObjects returns all world object ids
+  rpc ListWorldObjects(ListWorldObjectsRequest) returns (ListWorldObjectsResponse) {}
+
+  // GetWorldObject returns a world object by id
+  rpc GetWorldObject(GetWorldObjectRequest) returns (GetWorldObjectResponse) {}
+
+  // DoCommand sends/receives arbitrary commands
+  rpc DoCommand(common.v1.DoCommandRequest) returns (common.v1.DoCommandResponse) {
+    option (google.api.http) = {post: "/viam/api/v1/service/worldobjectstore/{name}/do_command"};
+  }
+}
+
+message ListWorldObjectsRequest {
+  // Name of the world object store service
+  string name = 1;
+
+  // Additional arguments to the method
+  google.protobuf.Struct extra = 99;
+}
+
+message ListWorldObjectsResponse {
+  repeated bytes uuids = 1;
+}
+
+message GetWorldObjectRequest {
+  // Name of the world object store service
+  string name = 1;
+
+  bytes uuid = 2;
+
+  // Additional arguments to the method
+  google.protobuf.Struct extra = 99;
+}
+
+message GetWorldObjectResponse {
+  common.v1.WorldState world_state = 1;
```

Comment:
@micheal-parks per Eliot's note I updated this, which is defined in the [common protos](https://github.com/viamrobotics/api/blob/main/proto/viam/common/v1/common.proto#L152)